interfaces/repo_test.go

```diff
@@ -43,26 +44,71 @@ var _ = Suite(&RepositorySuite{
 })
 
 func (s *RepositorySuite) SetUpTest(c *C) {
-	s.plug = &Plug{
-		Snap:      "provider",
-		Name:      "plug",
-		Interface: "interface",
-		Label:     "label",
-		Attrs:     map[string]interface{}{"attr": "value"},
-		Apps:      []string{"meta/hooks/plug"},
-	}
-	s.slot = &Slot{
-		Snap:      "consumer",
-		Name:      "slot",
-		Interface: "interface",
-		Label:     "label",
-		Attrs:     map[string]interface{}{"attr": "value"},
-		Apps:      []string{"app"},
-	}
+	// NOTE: the names provider/consumer are confusing. They will be fixed shortly.
+	provider, err := snap.InfoFromSnapYaml([]byte(`
+name: provider
+apps:
+    meta/hooks/plug:
```

Comment:
Can this please be changed? The test does not depend on this name, and this is not what we want to do about hooks I think.